prov/efa/src/rdm/rxr_rma.c

```diff
@@ -155,6 +155,10 @@ ssize_t rxr_rma_readmsg(struct fid_ep *ep, const struct fi_msg_rma *msg, uint64_
 	       flags);
 
 	rxr_ep = container_of(ep, struct rxr_ep, base_ep.util_ep.ep_fid.fid);
+	if ( OFI_UNLIKELY(0 == rxr_ep_domain(rxr_ep)->info->caps & FI_RMA) ) {
```

Comment:
Since it is a common util, should we extract it to static inline?

Reply:
Done: `rxr_ep_cap_check_rma` and `rxr_ep_cap_check_atomic`